util/cron/nightly

```diff
@@ -179,35 +182,36 @@ if (@ARGV) {
-    print "\t-no-local    : run tests using --no-local\n";
-    print "\t-componly    : only run the compiler, not the generated binary\n";
-    print "\t-futures     : run all futures, not just those with skipifs\n";
-    print "\t-no-futures  : do not run future tests\n";
-    print "\t-parnodefile : specify a node file to use for parallel testing\n";
-    print "\t-cron-recipient : send -cron emails to this address instead of SourceForge mailing list(s)\n";
-    print "\t-junit-xml   : create jUnit XML style report (default is \"on\" in Jenkins environment)\n";
+    print "\t-debug                         : check out sources and run for individual user (default)\n";
+    print "\t-cron                          : use for nightly cron runs only\n";
+    print "\t-notest                        : don't run the tests (check the build only)\n";
+    print "\t-noruntime                     : don't build the runtime or run the tests\n";
+    print "\t-examples                      : run the release/examples tests only\n";
+    print "\t-localeModel                   : run localeModel suite only\n";
+    print "\t-multilocale                   : run multilocale suite only\n";
+    print "\t-hello                         : run the release/examples/hello.chpl test only\n";
+    print "\t-hellos                        : run the release/examples/hello*.chpl tests only\n";
+    print "\t-valgrind                      : run tests in valgrind mode\n";
+#    print "\t-interpret                    : run tests in interpreted mode\n";
+    print "\t-performance                   : run performance tests\n";
+    print "\t-performance-description       : run performance tests with additional description\n";
+    print "\t-performance-configs           : comma seperated configs to graph, ': v' after config to be visible by default\n";
+    print "\t-compperformance <description> : run tests with compiler performance tracking\n";
+    print "\t-numtrials <number>            : number of trials to run\n";
+    print "\t-startdate                     : run performance tests providing a common start date to all the graphs\n";
+    print "\t-baseline                      : run testins using --baseline\n";
+    print "\t-dist <dist>                   : run distribution robustness tests\n";
+    print "\t-fast                          : run tests using --fast\n";
+    print "\t-llvm                          : run tests using --llvm\n";
+    print "\t-memleaks <log>                : run memleaks tests\n";
+    print "\t-no-local                      : run tests using --no-local\n";
+    print "\t-componly                      : only run the compiler, not the generated binary\n";
+    print "\t-futures                       : run all futures, not just those with skipifs\n";
+    print "\t-no-futures                    : do not run future tests\n";
+    print "\t-no-buildcheck                 : do not run checkChplInstall before running tests\n";
```

Comment:
Really minor, but maybe update this to "do not run `make check` before running tests\n" ?